templates/whatson/event_page.html

```diff
@@ -43,7 +43,17 @@
                                 <i class="fa-solid fa-location-dot event-meta__icon"></i>
 
                                 <div class="event-meta__details">
-                                    <p>{{ page.get_venue_type_display }}</p>
+                                    {% if page.venue_type == "in_person" and page.venue_address %}
+                                        {% if page.venue_website %}
+                                            <a class="event-details-banner__link" href="{{ page.venue_website }}">
+                                        {% endif %}
+                                                {{ page.venue_address|process_address|richtext }}
```

Comment:
Do we have a link to a website where the link text is an address?

Reply:
Yes according to the feedback from Ashley here: https://national-archives.atlassian.net/browse/DSWO-46?focusedCommentId=60823 " If Venue type is set as ‘In person’ the ‘Venue address’ shown in the Key details bar should also link to ‘Venue website’."